beerus-rpc/tests/utils.rs

```diff
@@ -0,0 +1,33 @@
+use beerus_core::{
+    config::Config,
+    lightclient::{
+        beerus::BeerusLightClient, ethereum::MockEthereumLightClient,
+        starknet::StarkNetLightClientImpl,
+    },
+};
+use beerus_rpc::{beerus_rpc_server::BeerusRpc, utils::wiremock::setup_wiremock};
+use std::path::PathBuf;
+
+pub const DEFAULT_DATA_DIR: &str = "/tmp";
+
+pub async fn setup_beerus_rpc() -> BeerusRpc {
+    let mock_starknet_rpc = setup_wiremock().await;
+    let config = Config {
+        ethereum_network: "goerli".to_string(),
+        ethereum_consensus_rpc: "".to_string(),
+        ethereum_execution_rpc: "".to_string(),
+        starknet_rpc: mock_starknet_rpc,
+        starknet_core_contract_address: Default::default(),
+        data_dir: Some(PathBuf::from(DEFAULT_DATA_DIR)),
+    };
```

Comment:
It needs to be validated, the config has several attributes that I didn't know how to fill, so they were left blank.

Reply:
Maybe the `new_from_env` method might be helpful to instantiate a mutable `Config` object and then you just pass in the mock starknet rpc

Reply:
Thanks @ftupas! I made a small refactor and I liked more how it looks now, but feedback is always welcome :smiley: